respec/Inleiding.md

```diff
@@ -1,21 +1,36 @@
+
+De elementen in deze standaard, begrippenkader, begrip en collectie, worden in 3 stappen beschreven:
+- Bij ieder begrip in deze standaard starten we met techniekonafhankelijke principes voor het beschrijven van een begrippenkader, de overeengekomen richtlijnen.
+- Iedere eigenschap van een begrip beschrijven we inhoudelijk met een definitie, uitleg, toelichting, één of meerdere redactionele notities en eventueel een voorbeeld.
+- Daarnaast beschrijven we per eigenschap van een begrip of deze verplicht/aanbevolen/optioneel is en hoe kan worden gevalideerd dat deze beschrijving aan de standaard voldoet. Hiermee bedoelen we:
+  - Verplicht: je voldoet (alleen) aan het profiel als voldaan is aan de voorwaarden voor het gebruik van deze eigenschap;
+  - Aanbevolen: je voldoet aan het profiel, zelfs als je geen gebruik maakt van deze eigenschap. Wel wordt het gebruik aanbevolen en zul je moet uitleggen waarom je geen gebruik maakt van deze eigenschap;
+  - Optioneel: je voldoet aan het profiel, ook als je geen gebruik maakt van deze eigenschap.
+
+Uitbreidingen van het basisprofiel zijn altijd optioneel.
+
+Voor het beschrijven van het profiel maken we gebruik van bestaande W3C standaarden. Dit leidt tot een abstracte syntax van het profiel uitgedrukt in de SHACL standaard. Daarnaast geven we een taalbinding van deze abstracte syntax naar andere bestaande W3C standaarden, zoals SKOS en DublinCore. Voor de abstracte syntax gebruiken we URI's die uitgedrukt worden in de Engelse taal, om zo direct een internationale aansluiting te houden. De abstracte syntax zal daarnaast Nederlandse labels kennen die in de deze lopende (Nederlandstalige) tekst zal worden gebruikt. De meerwaarde van het gebruik van SHACL is dat deze praktische handvatten biedt bij het toepassen van deze bestaande standaarden, inclusief een validatiemogelijkheid. Door het geven van de taalbinding naar de andere W3C standaarden bieden we een profiel dat direct kan worden begrepen door de internationale (internet) gemeenschap. Specifiek voor de Nederlandse (overheids)situatie geldt dat we hierbij aansluiten op de standaarden zoals opgenomen op de Pas-Toe-Of-Leg-Uit lijst conform de Nederlandse Overheids Referentie Architectuur (NORA).
 
 ### Algemene overeengekomen richtlijnen
 Om het profiel succesvol te laten zijn dient het breed gedragen te zijn. Om dit te realiseren hebben we een breed scala aan vertegenwoordigers samen gesteld vanuit diverse (semi-overheid)instanties. Vanuit deze werkgroep hebben we een aantal richtlijnen opgesteld voor het profiel. Deze lijst is niet uitputtend en volledig, maar dient als een basis.
```

Comment:
Ik las "deze lijst" als deze lijst van vertegenwoordigers. Beter: "onderstaande lijst".
Of misschien nog beter: een aparte sectie over de samenstelling van de werkgroep en de gevolgde werkwijze. Hoeft niet heel formeel, maar er mag best iets meer toegelicht worden hoe het profiel toe stand is gekomen.

Reply:
klopt, wat ambigu. Ik heb de ambiguïteit nu opgelost door de term "lijst" te vermijden.